toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -728,16 +727,18 @@ func (c *Chroot) stopGPGComponents() (err error) {
 		return
 	}
 
-	_, err = exec.LookPath("gpgconf")
-	if err != nil {
-		logger.Log.Debugf("gpgconf is not installed, so gpg-agent is not running: %s", err)
-		return nil
-	}
-
 	err = c.UnsafeRun(func() (err error) {
-		components, err := listGPGComponents()
-		if err != nil {
-			return err
+		found, chrootErr := file.CommandExists("gpgconf")
+		if chrootErr != nil {
+			return chrootErr
+		} else if !found {
+			logger.Log.Debugf("gpgconf is not installed, so gpg-agent is not running: %s", err)
+			return nil
+		}
```

Comment:
Small suggestion: since we already exit when we find an error, we can drop `else`. And I think in most cases we follow the logic of a separate error check + exit and then separately writing the happy path logic.
```suggestion
		}
		
		if ! found {
			logger.Log.Debugf("gpgconf is not installed, so gpg-agent is not running: %s", err)
			return nil
		}
```